sources/packages/web/src/views/aest/institution/ProgramDetails.vue

```diff
@@ -9,7 +9,7 @@
       subTitle="View program"
     >
       <template #buttons>
-        <div v-if="isPendingProgram">
+        <v-row class="p-0 m-0" v-if="isPendingProgram">
```

Comment:
Please refer to my previous comment on the use of this class.

Reply:
commented above